sharktank/sharktank/utils/export_artifacts.py

```diff
@@ -356,13 +355,8 @@ def export_llm_to_mlir(
             export_args.append("--use-attention-mask")
         if self.use_hf:
             export_args.append("--use-hf")
-
-        self._run_cmd(
-            cmd=subprocess.list2cmdline(export_args),
-            run_msg="Exporting MLIR",
-            success_msg="Exported to MLIR successfully",
-            exception=ExportMlirException,
-        )
+        sys.argv = export_args
+        export_paged_llm_v1.main()
```

Comment:
We shouldn't do this. If there is an infinite loop or the program hands, we should find out why. This works for both llama and deepseek

Reply:
`capture_output=True` is needed in order to use the xfail that matches on a string.